requirements/dev.txt

```diff
@@ -1 +1,2 @@
 tox==4.12.1
+GitPython==3.1.43
```

Comment:
You need it in dev because you need it to run the setup script. Though you may not need it forever, so perhaps I can add removing this line programmatically in the setup script.

Reply:
but if you use the tox test environment to run the script, that will have all the dependencies (except tox itself). Anyhow, no big deal

Reply:
Right, I mean the person acutally using the template needs it. Would they get it another way?

Reply:
I think this should be here, if we need it - we need it. I'm not a fan of us making the deps asymmetric such that some exist only in the requirements files and some only elsewhere.